### PR TITLE
TDL-18688: Fix replication key for the roles stream

### PR DESCRIPTION
**Pull request by hpatel41**
# Description of change
TDL-18688: Fix replication key for the roles stream
- Made Roles stream as FULL_TABLE stream, as we are not getting **updated_at** from the API.

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tap_snapchat_ads/streams.py [hunk 1]
```diff
@@ -70,8 +70,7 @@
             # Reference: https://developers.snapchat.com/api/docs/?python#get-all-roles-in-organization
             'roles': {
                 'key_properties': ['id'],
-                'replication_method': 'INCREMENTAL',
-                'replication_keys': ['updated_at'],
+                'replication_method': 'FULL_TABLE', # made 'Roles' stream as FULL_TABLE stream, as we are not getting 'updated_at' from the API response
```
Comment (dbshah1212) [inline tap_snapchat_ads/streams.py:73]: Remove comments and add inline comments. Do we need to change integration tests?

Comment (harshpatel4crest) [reply tap_snapchat_ads/streams.py]: No, we need to update the integration tests after merging the Tap-Tester tests PR

File: tap_snapchat_ads/streams.py [hunk 2]
```diff
@@ -70,8 +70,7 @@
             # Reference: https://developers.snapchat.com/api/docs/?python#get-all-roles-in-organization
             'roles': {
                 'key_properties': ['id'],
-                'replication_method': 'INCREMENTAL',
-                'replication_keys': ['updated_at'],
+                'replication_method': 'FULL_TABLE',
```
Comment (dbshah1212) [inline tap_snapchat_ads/streams.py:73]: @harshpatel4crest Made 'Roles' stream as FULL_TABLE stream, as we are not getting 'updated_at' from the API response, right?

Comment (harshpatel4crest) [reply tap_snapchat_ads/streams.py:73]: Yes, adding comment.

**Comment by harshpatel4crest** [comment]
The **Roles** stream is INCREMENTAL in PR #15, we need to update it at the time of merging.

**Review by savan-chovatiya — APPROVED** [review]

**Review by dbshah1212 — APPROVED** [review]

**Review by RushiT0122 — APPROVED** [review]

**Review by KrisPersonal — APPROVED** [review]

**Review by kspeer825 — APPROVED** [review]
Assuming this change will be covered by tests other PR

**Pull request merged**